lms/djangoapps/static_template_view/views.py

```diff
@@ -46,7 +46,11 @@ def render(request, template):
     content_type, __ = mimetypes.guess_type(template)
 
     try:
-        return render_to_response('static_templates/' + template, {}, content_type=content_type)
+        context = {}
+        # This is necessary for the dialog presented with the TOS in /register
+        if template == 'honor.html':
+            context['allow_iframing'] = True
```

Comment:
@douglashall It allows the /honor view to be displayed inside the modal iframe. Otherwise [this part of the main.html template](https://github.com/edx/edx-platform/blob/master/lms/templates/main.html#L44-L53) redirects to the full /honor URL.